students/kyle_lehning/Lesson06/mailroom_part4.py

```diff
@@ -116,13 +116,23 @@ def print_all():
     file_location = tempfile.gettempdir()
     print("letters stored at: " + file_location)
     for key, value in donors.items():
-        file_name = "{}{}{}{}{}".format(file_location, "\\", value["name"], time.strftime("%Y%m%d-%H%M%S"), ".txt")
-        with open(file_name, "w") as file:
-            file.write("Dear {name},"
-                       "\n\nThank you for your very kind donations totaling ${total_don:.2f}."
-                       "\nIt will be put to very good use."
-                       "\n\nSincerely,"
-                       "\n-The Team".format(**value))
+        file_path_name = "{}{}{}".format(file_location, "\\", value["name"])
+        file = file_creation(file_path_name)
+        file.write(thank_you_text(value))
+        file.close()
+
+
+def file_creation(file_name):
+    return open("{}{}{}".format(file_name, time.strftime("%Y%m%d-%H%M%S"), ".txt"), "w")
```

Comment:
This isn't working for me - are you getting actual files created?

Reply:
Yes this creates files for me. It also has been passing the tests for it. Does it print to the window where it should be creating it when send all is selected?